src/lib/viewers/doc/DocBaseViewer.js

```diff
@@ -846,17 +850,17 @@ class DocBaseViewer extends BaseViewer {
      * @return {void}
      */
     pagechangeHandler(event) {
-        const pageNum = event.pageNumber;
-        this.pageControls.updateCurrentPage(pageNum);
+        const pageNumber = event.pageNumber;
+        this.pageControls.updateCurrentPage(pageNumber);
 
         // We only set cache the current page if 'pagechange' was fired after
         // preview is loaded - this filters out pagechange events fired by
         // the viewer's initialization
         if (this.loaded) {
-            this.cachePage(pageNum);
```

Comment:
Are we going to add this similar functionality to the multiimage viewer?

Reply:
that's the plan!